pkg/resource/template/fileStat_posix.go

```diff
@@ -18,13 +18,15 @@ func fileStat(name string) (fi fileInfo, err error) {
 		if err != nil {
 			return fi, err
 		}
-		defer f.Close()
+		defer f.Close() // nolint: errcheck
 		stats, _ := f.Stat()
```

Comment:
Why don't we handle the error case here?

Reply:
i thought about doing it but then to me it looked like that `fi` was valid in the non-error return case and returning the `f.Close()` error wouldn't be useful.
At least that's what i was thinking.
Will change it to `f.Close()` error

Reply:
no not the defer line.... the f.Stat() returns an error.  If that returns an error shouldn't we propagate.